src/plugins/vis_type_timeseries/public/application/components/index_pattern.js

```diff
@@ -229,6 +255,74 @@ export const IndexPattern = ({ fields, prefix, onChange, disabled, model: _model
           </EuiFormRow>
         </EuiFlexItem>
       </EuiFlexGroup>
+      {allowLevelofDetail && (
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiFormRow
+              id={htmlId('detailLevel')}
+              label={
+                <>
+                  <FormattedMessage
+                    id="visTypeTimeseries.indexPattern.detailLevel"
+                    defaultMessage="Level of detail"
+                  />{' '}
+                  <EuiIconTip
+                    position="right"
+                    content={
+                      <FormattedMessage
+                        id="visTypeTimeseries.indexPattern.detailLevelHelpText"
+                        defaultMessage="Intervals will be selected automatically based on the current timeframe and configuration property {histogramMaxBars}."
```

Comment:
I'd like to update this text so that it also includes some more actionable info, and also mentions the target bars:

```suggestion
                        defaultMessage="Controls the auto interval based on the time range. The default interval is affected by the advanced settings {histogramTargetBars} and {histogramMaxBars}."
```